grpc/src/main/java/com/linecorp/armeria/internal/grpc/ArmeriaMessageDeframer.java

```diff
@@ -379,6 +385,42 @@ private void readHeader() {
         state = State.BODY;
     }
 
+    private long readHeaderAsLong() {
+        unprocessedBytes -= HEADER_LENGTH;
+        assert unprocessed != null;
+        final ByteBuf firstBuf = unprocessed.peek();
+        assert firstBuf != null;
+        final int firstBufLen = firstBuf.readableBytes();
+
+        if (firstBufLen >= HEADER_LENGTH) {
+            final long header = (firstBuf.readUnsignedInt() << 8) | firstBuf.readUnsignedByte();
+            if (firstBufLen == HEADER_LENGTH) {
```

Comment:
Probably a bit clearer to use `firstBuf.isReadable` like below

Reply:
Fixed